lighthouse-core/report/templates/report-template.html

```diff
@@ -247,8 +249,14 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
```

Comment:
Thinking about this some more, I'm not stoked on moving this:

```
window.addEventListener('DOMContentLoaded', _ => {
  window.logger = new Logger('#log');
  new LighthouseReport(window.lhresults);
});
```

into a separate script. There's little value in doing that. Plus, it'll need a license header and a bunch of eslint ignore boilerplate. All of that will add to our browserified bundles.

Next step is to kill some of that extra stuff in https://github.com/GoogleChrome/lighthouse/issues/1670